lib/stm32/f4/pwr.c

```diff
@@ -38,9 +38,15 @@
 
 void pwr_set_vos_scale(enum pwr_vos_scale scale)
 {
+	/*
+	 * This function only works fully for F405xx/07xx and F415xx/17xx.
+	 * Scale 1 mode is power saving and is the default at reset.
+	 * Scale 2 mode is high performance.
```

Comment:
I can change these to read simply "Scale 1 mode is the default at reset." if needed, as I am unclear as to which is which.